validation_pipeline.ipynb

```diff
@@ -25,12 +25,14 @@
         "DEBUG = False #@param {type:\"boolean\"}\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    mean_rmse, var_rmse, overall_rmse = evaluation.calculate_rmse(eval_dataset, means_isoscape, vars_isoscape, MEAN_TRUTH_NAME, VAR_TRUTH_NAME, MEAN_PREDICTED_NAME, VAR_PREDICTED_NAME)
</pre><p>Are we averaging the RMSE over all our isoscapes? Is this just for Oxygen? Clarification needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Just oxygen. That's the RMSE we measure for the current model we're trying to build.

Reply:
That makes sense. If this is meant for end-to-end evaluation though, it should ideally do the same thing for each isoscape, per our [isoscape validation design doc](go/ddf-isoscape-validation)